docs/docs/tutorials/get-started/README.mdx

```diff
@@ -10,7 +9,7 @@ import gitpodRunning from './assets/gitpod-running.png';
 
 Prepend `https://gitpod.io/#` to the Logto repo URL (or <a href="https://gitpod.io/#https://github.com/logto-io/logto" target="_blank">click here</a>) and access it. Wait a few moment, you'll see the message like:
 
-<img src={gitpodRunning} alt="GitPod Running" width="600px" />
+![Gitpod is running](./assets/gitpod-running.png)
```

Comment:
what's the feeling of this? it was by design since the screenshot was too large. i saw the new one is even larger

Reply:
updated. PTAL